src/utils/prim-lexing.fs

```diff
@@ -201,7 +201,7 @@ namespace Internal.Utilities.Text.Lexing
         let numUnicodeCategories = 30 
         let numLowUnicodeChars = 128 
         let numSpecificUnicodeChars = (trans.[0].Length - 1 - numLowUnicodeChars - numUnicodeCategories)/2
-        let lookupUnicodeCharacters (state,inp) = 
+        let lookupUnicodeCharacters state inp =
```

Comment:
@vasily-kirichenko Could you remove the changes in ``prim-lexing.fs`` please? I'm pretty sure they don't remove  any allocations. (If they do then let's discuss further, there must be something I'm missing).  Thanks!!